### ISSUE TITLE
TAStudio New From Current SaveRam cause Exception: "Sequence contains no elements"

### ISSUE THREAD

**Issue by Croopper** [TASVideos/BizHawk#1541]
This movie shows problem:
[bh_ex.zip](https://github.com/TASVideos/BizHawk/files/3078868/bh_ex.zip)

**Comment by YoshiRulz** [comment]
Reproducible with EmuHawk 2.3.1 on Win10 *Home* 1709.

```
System.InvalidOperationException: Sequence contains no elements
   at System.Linq.Enumerable.First[TSource](IEnumerable`1 source)
   at BizHawk.Client.EmuHawk.TAStudio.StartANewProjectFromSaveRamMenuItem_Click(Object sender, EventArgs e)
   at System.Windows.Forms.ToolStripItem.RaiseEvent(Object key, EventArgs e)
   at System.Windows.Forms.ToolStripMenuItem.OnClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleClick(EventArgs e)
   at System.Windows.Forms.ToolStripItem.HandleMouseUp(MouseEventArgs e)
   at System.Windows.Forms.ToolStrip.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.ToolStripDropDown.OnMouseUp(MouseEventArgs mea)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ToolStrip.WndProc(Message& m)
   at System.Windows.Forms.ToolStripDropDown.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

Looks to be here:
https://github.com/TASVideos/BizHawk/blob/e6374ef4771ff0f0b973bd5f671675269e3f7a96/BizHawk.Client.EmuHawk/tools/TAStudio/TAStudio.MenuItems.cs#L1482

**Comment by alyosha-tas** [comment]
That's strange. Is it unique to mGBA? I tried on GB and it doesn't seem to happen.

**Comment by YoshiRulz** [comment]
I used Mupen64Plus.

**Comment by alyosha-tas** [comment]
I can't reproduce this in the dev build. please try that:

https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts

**Comment by YoshiRulz** [comment]
Confirmed fixed on or before 7915ec8bc.

**Comment by alyosha-tas** [comment]
i fixed the above exception
